connector/datadogconnector/config_test.go

```diff
@@ -30,14 +29,14 @@ func TestValidate(t *testing.T) {
 			cfg: &Config{Traces: TracesConfig{
 				SpanNameRemappings: map[string]string{"oldname": ""},
 			}},
-			err: "\"\" is not valid value for span name remapping",
+			err: "'' is not valid value for span name remapping",
 		},
 		{
 			name: "span name remapping empty key",
 			cfg: &Config{Traces: TracesConfig{
 				SpanNameRemappings: map[string]string{"": "newname"},
 			}},
-			err: "\"\" is not valid key for span name remapping",
+			err: "'' is not valid key for span name remapping",
```

Comment:
Why did this change?

Reply:
The error is different in connector & Exporter today 
https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/bd98b9d9b4226430744ccdbfe2ec202cd3237c72/exporter/datadogexporter/config.go#L496-L502
https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/bd98b9d9b4226430744ccdbfe2ec202cd3237c72/connector/datadogconnector/config.go#L101

I have used exporter error going forward.